Sources/MapboxMaps/Foundation/MapboxMap.swift

```diff
@@ -31,7 +31,7 @@ internal protocol MapboxMapProtocol: AnyObject {
     func camera(for coordinateBounds: CoordinateBounds, padding: UIEdgeInsets, bearing: Double?, pitch: Double?) -> CameraOptions
     func coordinate(for point: CGPoint) -> CLLocationCoordinate2D
     func point(for coordinate: CLLocationCoordinate2D) -> CGPoint
-    func performWithoutNotifying(_ block: () -> Void)
```

Comment:
is this main sdk change required by the swiftUI sdk? would it be possible to extend it in the swiftUI module?